packages/extension/src/companion/CompanionPermission.tsx

```diff
@@ -0,0 +1,85 @@
+import { Button } from '@dailydotdev/shared/src/components/buttons/Button';
+import classed from '@dailydotdev/shared/src/lib/classed';
+import React, { ReactElement, Ref, forwardRef, useContext } from 'react';
+import PlayIcon from '@dailydotdev/shared/icons/filled/play.svg';
+import FeaturesContext from '@dailydotdev/shared/src/contexts/FeaturesContext';
+import {
+  Features,
+  getFeatureValue,
+} from '@dailydotdev/shared/src/lib/featureManagement';
+import { useExtensionPermission } from './useExtensionPermission';
+
+interface Content {
+  title: string;
+  description: string;
+}
+
+const contentVariation: Record<string, Content> = {
+  v1: {
```

Comment:
I thought it would be fixed values from this image? Though I agree it would be much more flexible if we won't restrict it by version.
![image](https://user-images.githubusercontent.com/13744167/169927350-7904d488-a46a-4019-a915-29ce6717e163.png)

Reply:
I mean generally when we roll out AB test we create a string variable for the title with V1 as the default.
But perhaps @idoshamun can shed some light on how he likes to proceed.

Reply:
I have now specified different variables for title and description to have more flexibility on how we would want it to be AB tested. Thank you for the thoughts!

Reply:
I agree with @rebelchris. The version is for the layout not the copy

Reply:
Cool. The current implementation should support two different variables for title and description copy.